python/lsst/drp/tasks/__init__.py

```diff
@@ -19,5 +19,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+# TODO: do we put a note that this file is intentionally empty?
+
```

Comment:
Is it empty?

Reply:
Any resolution on this bare TODO?

Reply:
I'll ask on slack, because I have no idea. I just noted something that I thought was strange.